frontend/zc_messaging/src/pages/message-board/index.jsx

```diff
@@ -170,11 +173,28 @@ const MessagingBoard = () => {
         emoji: emoji,
         reactedUsersId: [currentUserId]
       }
-      message.emojis.push(newEmojiObject)
+      let emojisArray = [...message.emojis, newEmojiObject]
+      // message.emojis = emojisArray
```

Comment:
If the code isn't being used, you can do away with it entirely. Same applies to the rest of this file. If there's any code that you commented out that you know cannot be used in the future, do away with it